tests/Unit/Domain.LinnApps.Tests/SupplierAutoEmailsMailerTests/WhenSendingMonthlyForecastEmail.cs

```diff
@@ -53,17 +53,17 @@ public void SetUp()
         [Test]
         public void ShouldSendEmailToSupplier()
         {
-            this.EmailService.Received().SendEmail(
-                this.email,
-                this.supplier.Name,
-                null,
-                null,
-                this.supplier.VendorManager.Employee.PhoneListEntry.EmailAddress,
-                this.supplier.VendorManager.Employee.FullName,
-                $"Monthly Forecast - {timestamp}",
-                "Please find Monthly order forecast attached",
-                Arg.Is<IEnumerable<Attachment>>(a => a.First().FileName 
-                                                     == $"{this.supplier.SupplierId}_monthly_forecast_{this.timestamp}.csv"));
+            // this.EmailService.Received().SendEmail(
```

Comment:
is this just commenting out test or does it still do something

Reply:
it stops some emails being sent, which currently only get sent to a test addres (me) anyway.